src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,28 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import org.hibernate.SessionFactory;
 
 public class MovieServiceImpl implements MovieService {
+    private MovieDao movieDao;
+
+    public MovieServiceImpl(SessionFactory sessionFactory) {
+        this.movieDao = new MovieDaoImpl(sessionFactory);
+    }
```

Comment:
```suggestion
    public MovieServiceImpl(MovieDao movieDao) {
        this.movieDao = movieDao;
    }
```
it is better to pass an instance of daoService
same with other services